src/tests/test_utils/functional_test_utils/include/functional_test_utils/skip_tests_config.hpp

```diff
@@ -11,6 +11,7 @@
 #include <vector>
 
 std::vector<std::string> disabledTestPatterns();
+bool isGPU1Present();
```

Comment:
I think is not required to declare yhis function in .hpp as it has only single file local scope.